components/system-broker/cmd/main.go

```diff
@@ -62,11 +61,10 @@ func main() {
 	directorGraphQLClient, err := prepareGqlClient(cfg, uuidSrv)
 	fatalOnError(err)
 
-	systemBroker := osb.NewSystemBroker(directorGraphQLClient, cfg.Server.SelfURL)
+	systemBroker := osb.NewSystemBroker(directorGraphQLClient, cfg.Server.SelfURL+cfg.Server.RootAPI)
```

Comment:
im not sure if this will be ok for the middlewares registered in the osb/api.go file. They have to run only on broker calls and not when trying to fetch specs or hitting the health api while the filters that are registered during server creation should run on all api calls (osb+specs +health) (TODO should be verified)

Reply:
Previously SelfURL had RootAPI in it which is redundant since we have a separate configuration for the RootAPI and we can reuse it. In the end this is the same URL but crafted from two unique variables and not having double maintanance of the RootAPI one separate and one part of SelfURL.